apps/marketplace/components/Opportunities/OpportunitiesFilters.js

```diff
@@ -8,7 +8,6 @@ import { changeForm, loadOpportunities } from 'marketplace/actions/opportunities
 import CheckboxDetailsField from 'shared/form/CheckboxDetailsField'
 import AUaccordion from '@gov.au/accordion/lib/js/react.js'
 import AUheading from '@gov.au/headings/lib/js/react.js'
-import TypeFilterControl from './TypeFilterControl'
```

Comment:
You should be able to delete the `TypeFilerControl.js` file and the `a.filter` and `a:hover.filter, a.filter.active` rules in `Opportunities.scss`.

Reply:
And the `li.filterContainer` rule.